modules/generator/registry/histogram.go

```diff
@@ -149,7 +149,7 @@ func (h *histogram) name() string {
 	return h.metricName
 }
 
-func (h *histogram) collectMetrics(appender storage.Appender, timeMs int64, externalLabels map[string]string) (activeSeries int, err error) {
+func (h *histogram) collectMetrics(appender storage.Appender, timeMs int64, externalLabels map[string]string, traceIDLabelName string) (activeSeries int, err error) {
```

Comment:
does it make sense to add this as a member of the struct instead of passing it in every time? or do we pass it in every time to handle dynamically changing values through overrides?

Reply:
good idea - changed